src/pages/Activity/ActivityCard.tsx

```diff
@@ -8,44 +8,42 @@ export function ActivityCard({ content }: ActivityCardProps) {
   return (
     <Box
       key={content.id}
-      border="1px solid"
-      borderColor="grey.200"
-      padding={3}
-      borderRadius={3}
+      sx={{
+        border: "1px solid",
+        borderColor: "grey.300",
+        borderRadius: 2,
+        p: 2,
+        backgroundColor: "white",
+        boxShadow: "0px 1px 4px rgba(0,0,0,0.1)",
+        display: "flex",
+        flexDirection: "column",
+        gap: 1.5,
+      }}
     >
-      <Box mb={1} display="flex" justifyContent="space-between">
-        <Typography fontWeight="bold" variant="subtitle1">
+      <Box display="flex" justifyContent="space-between" alignItems="center">
+        <Typography variant="subtitle1" fontWeight="bold" color="teal">
           {content.name}
         </Typography>
-        <Button variant="contained" sx={{ aspectRatio: "4/4", padding: 0.5 }}>
-          <Edit sx={{ width: 0.75 }} />
-        </Button>
-      </Box>
-
-      <Box display="grid" gridTemplateColumns="1fr 2fr" gap={0.5}>
-        <Typography variant="subtitle2" fontWeight="bold">
-          {strings.activityList.card.area}
-        </Typography>
-        <Typography variant="subtitle2" fontWeight="bold" color="grey.600">
-          {content.area}
-        </Typography>
-      </Box>
-      <Box display="grid" gridTemplateColumns="1fr 2fr" gap={0.5}>
-        <Typography variant="subtitle2" fontWeight="bold">
-          {strings.activityList.card.frequency}
-        </Typography>
-        <Typography variant="subtitle2" fontWeight="bold" color="grey.600">
-          {content.frequency}
-        </Typography>
-      </Box>
-      <Box display="grid" gridTemplateColumns="1fr 2fr" gap={0.5}>
-        <Typography variant="subtitle2" fontWeight="bold">
-          {strings.activityList.card.teacher}
-        </Typography>
-        <Typography variant="subtitle2" fontWeight="bold" color="grey.600">
-          {content.teacher}
-        </Typography>
+        <IconButton size="small" color="primary">
+          <Edit fontSize="small" />
+        </IconButton>
       </Box>
+      <Button
+        variant="outlined"
+        fullWidth
+        sx={{
+          borderColor: "teal",
+          color: "teal",
```

Comment:
No loose colors
Talvez não precisemos especificar as cores aqui, ver comentário a seguir
```suggestion
          borderColor: "primary.main",
          color: "primary.main",
```

Reply:
adoraria viver no incrivel mundo sem ages iii